desktop/cmp/rest/RestFormModel.js

```diff
@@ -40,25 +46,21 @@ export class RestFormModel {
         return this.controlModels.every(it => it.isValid);
     }
 
-    @computed
-    get isWritable() {
-        const {isAdd, actionEnabled} = this;
-        return (isAdd && actionEnabled.add) || (!isAdd && actionEnabled.edit);
-    }
-
     @computed
     get isDirty() {
         return !isEqual(this.record, this.originalRecord);
     }
 
-    constructor({parent, editors}) {
+    constructor({parent, editors, toolbarActions}) {
```

Comment:
I think we should consider making all of these RestXXX classes private for the time being, with the only public entry point being RestGridModel and RestGrid. 

If we did this, then the toolbarActions property would not have a default here, but just get its default from RestGridModel